boards/arm/actinius_icarus/dts/bindings/actinius,charger-enable.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Actinius B.V.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The Icarus IoT Board v2 provides the user with an option to disable the
+  charger or to leave it on auto. The option can be changed by changing
+  the `charger` property in the `charger_enable` dt node to `auto`
```

Comment:
compatibles should not reference hardcoded node names?

Reply:
Updated with generic wording, thanks!